<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="settingsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="xRotationLabel">
         <property name="text">
          <string>x rotation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="xSlider">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="yRotationLabel">
         <property name="text">
          <string>y rotation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="ySlider">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="zRotationLabel">
         <property name="text">
          <string>z rotation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="zSlider">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="vSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="shaderLabel">
         <property name="text">
          <string>Renderer</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="shaderSelect">
         <item>
          <property name="text">
           <string>Flat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Goraud</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phong</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Toon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Textured</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Deferred blitted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Deferred</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="modelLabel">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="modelSelect">
         <item>
          <property name="text">
           <string>cube.obj</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sphere.obj</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>flat_surface.obj</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cat.obj</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="scaleLabel">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="scaleSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3000</number>
         </property>
         <property name="singleStep">
          <number>30</number>
         </property>
         <property name="pageStep">
          <number>300</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="mainView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
